### ISSUE TITLE
Logitech Keyboard Disconnects When Using Semi-Fisher

### ISSUE THREAD

**Issue by DarkAthena** [fishyboteso/fishyboteso#89]
Hi. I just started using fishybot and have noticed that my Logitech keyboard will occasionally disconnect while the bot is active. I'm currently using the semi-fisher mode. I have updated my windows 10 install, ESO, keyboard drivers, disabled all other addons, and I followed the fishybot install instructions exactly. Aside from the keyboard issue, the app works fine and it's wonderful!

Do you have any other suggestions for troubleshooting? Is this a known issue?

Thank you in advance for your help.

**Comment by SemjonKerner** [comment]
This happened for others. We could not pinpoint the reason so far and can't reproduce it on our end.

**Comment by DarkAthena** [comment]
Did anyone find a work-around?

**Comment by SemjonKerner** [comment]
Not really. Another user had some DIP switches on his keyboard that changed the behavior, but eventually it came back.

**Comment by adsau59** [comment]
@DarkAthena can u provide some hardware details/info of ur keyboard?

**Comment by DarkAthena** [comment]
Hi. I can. It's a Logitech G413. Full size keyboard. What else would you like to know?
